### ISSUE TITLE
Install on OSX

### ISSUE THREAD

**Issue by msageryd** [blackbeam/poppler-simple#25]
I'm having trouble building poppler-simple on OSX (10.14, Mojave)

Not sure what's essential information from the build log. Here is the first error:

```
../src/NodePopplerDocument.cc:34:12: error: no member named 'append' in 'GooList'
    pages->append((void *)p);
    ~~~~~  ^
```

**This is what I've done so far**:

`brew install poppler`
`brew install python2`
`brev install pkg-config`
`npm install poppler-simple`

**Comment by msageryd** [comment]
I tried to upgrade from Poppler 0.70 to 0.71, and got a different build error.

```
../src/NodePopplerPage.h:158:31: error: unknown type name 'GBool'
        TextPage *getTextPage(GBool rawOrder) {
```

**Comment by blackbeam** [comment]
Fixed in [v0.5.0](https://github.com/blackbeam/poppler-simple/releases/tag/v0.5.0)
